dbeam-core/src/main/java/com/spotify/dbeam/avro/JdbcAvroSchema.java

```diff
@@ -69,12 +68,16 @@ public static Schema createSchemaByReadingOneRow(
       throws SQLException {
     return createSchemaByReadingOneRow(
             connection, QueryBuilder.fromTablename(tablename),
-            avroSchemaNamespace, avroDoc, useLogicalTypes);
+            avroSchemaNamespace, avroDoc, useLogicalTypes, Optional.empty());
   }
 
```

Comment:
I believe the methods here should NOT be called when `inputSchema` is present. The goal of this class is to automatically generate an Avro Schema based on the Table schema. When the `inputSchema` is provided, there is no need to generate an Avro schema.

I might be missing something. Does it make sense?